src/main/java/org/futureCollars/lesson10/task3/ValidatorURL.java

```diff
@@ -0,0 +1,18 @@
+package org.futureCollars.lesson10.task3;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ValidatorURL {
+
+    public static void main(String[] args) {
+
+        Pattern pattern = Pattern.compile("http://www.onet.pl|https://mail.google.com|http://wiadmosci.onet.pl|http://onet.pl");
```

Comment:
ta sama uwaga co w poprzednim programie, napisz metodę która sprawdza czy podany parametr jest url czy nie. Zadaj sobie pytanie jak inny programista mógłby użyć twój kod żeby. Na tą chwilę musiałby napisać wszystko od nowa, a chodzi o to żeby twoje rozwiązanie było tak napisane, że będzie je można łatwo użyć w postaci np. metody